ams/server/pom.xml

```diff
@@ -55,12 +54,32 @@
             <groupId>com.netease.arctic</groupId>
             <artifactId>arctic-core</artifactId>
             <version>${project.version}</version>
+            <exclusions>
```

Comment:
Can we handle these exclude logics in the `core` module instead of requiring dependent parties to exclude them?